src/ToolBox/AsParameter.php

```diff
@@ -0,0 +1,14 @@
+<?php
+
+declare(strict_types=1);
+
+namespace PhpLlm\LlmChain\ToolBox;
```

Comment:
Do you agree with the namespace @chr-hertel ?

Reply:
what do you think about introducing an `Attribute` namespace? too early? rather below `ToolBox` or on a higher level?

Reply:
ToolBox/Attribute suits me

Reply:
Done